gridmet_split_script.py

```diff
@@ -0,0 +1,219 @@
+# Libraries
+import pickle
+import time
+import geopandas as gpd
+import geopandas.geodataframe
+import grd2shp_xagg
+import xagg as xa
+import xarray as xr
+import os
+import datetime
+
+# Variables
+
+## official list of variables needed for drb-inland-salinity model
+data_vars_shrt_all = ['tmmx', 'tmmn', 'pr', 'srad', 'vs','rmax','rmin','sph']
+## drb catchment polygons
+gdf_nhru02_path = './data/nhru_02/nhru_02.shp'
+gdf = gpd.read_file(gdf_nhru02_path)
+## date range
+start_date = '1979-01-01'
+end_date = '2021-01-01'
+## lat lon
+lon_min, lat_min, lon_max, lat_max = gdf.total_bounds
+## output folder
+output_path = './data/'
+
+
+def get_gridmet_datasets(variable, start_date, end_date, polygon_as_bbox = None, lon_min = None, lat_min = None, lon_max = None, lat_max = None):
+
+    """
+    :param str/list var_short: data variable short name or list of data variable short names. Must be one of the following: ['tmmx', 'tmmn', 'pr', 'srad', 'vs', 'rmax', 'rmin', 'sph']
+    :param str start_date: Start date of data collection yyyy-mm-dd
+    :param str end_date: End date of data collection yyyy-mm-dd
+    :param gdf.total_bounds polygon_bbox: total bounds of a geodataframe. Ex: geodataframe.total_bounds. If not known, None
+    :param str lon_min: bbox of aoi longitude min. None if polygon_bbox is given (i.e. polygon_bbox != None)
```

Comment:
I recommend changing this to match code:
```suggestion
    :param str lon_min: bbox of aoi longitude min. Not used if polygon_bbox is given (i.e. polygon_bbox != None)
```